$MirOS: ports/emulators/qemu/snapshot/patches/patch-bswap_h,v 1.1 2008/05/07 15:53:09 tg Exp $
--- bswap.h.orig	Wed Sep 23 19:01:07 2009
+++ bswap.h	Sat Oct 17 16:35:43 2009
@@ -49,20 +49,26 @@
 
 #endif /* !HAVE_BYTESWAP_H */
 
+#ifndef bswap16
 static inline uint16_t bswap16(uint16_t x)
 {
     return bswap_16(x);
 }
+#endif
 
+#ifndef bswap32
 static inline uint32_t bswap32(uint32_t x)
 {
     return bswap_32(x);
 }
+#endif
 
+#ifndef bswap64
 static inline uint64_t bswap64(uint64_t x)
 {
     return bswap_64(x);
 }
+#endif
 
 #endif /* ! HAVE_MACHINE_BSWAP_H */
 
